Automattic/newspack-newsletters
# feat: bootstrapping lists CPT

### All Submissions:


### Changes proposed in this Pull Request:

Creates a CPT to manage Subscription Lists

### How to test the changes in this Pull Request:

1. Check the code
2. Verify the Lists menu added under Newsletters
3. Try adding/editing Lists
4. Verify the content only shows a simple textarea

### Other information:
